include/mbedtls/mbedtls_config.h

```diff
@@ -3670,6 +3670,9 @@
 //#define MBEDTLS_ECP_WINDOW_SIZE            4 /**< Maximum window size used */
 //#define MBEDTLS_ECP_FIXED_POINT_OPTIM      1 /**< Enable fixed-point speed-up */
 
+/* RSA OPTIONS */
+//#define MBEDTLS_RSA_MIN_KEY_SIZE           1024 /**<  Minimum RSA key size allowed in bits (Minimum possible value is 128 bits) */
```

Comment:
More precisely, this is the minimum allowed for generation. It's still possible to import a smaller key.

So maybe the name should indicate that? (See also my comment about the PSA constant, which definitely needs a more precise name.) Something like `MBEDTLS_RSA_GEN_KEY_MIN_SIZE` given that the affected function is called `mbedtls_rsa_gen_key`.

Reply:
Agree, but maybe `MBEDTLS_RSA_GENERATE_KEY_MIN_SIZE` rather than `MBEDTLS_RSA_GEN_KEY_MIN_SIZE`?

Reply:
Also, we should enforce the 128 bit minimum via check_config.h

Reply:
Regarding the name: why not stick to something close to the name of the relevant function?

Regarding where to enforce the minimum: this would be better done in `rsa.h` or `rsa.c`. `check_config.h` is more suited to cross-module considerations.

Reply:
changed it to `MBEDTLS_RSA_GEN_KEY_MIN_SIZE` and added a check in rsa.h to check for 128 bits minimum, I removed the check from mbedtls_rsa_gen_key